Rakefile

```diff
@@ -3,17 +3,21 @@ require 'puppet-lint/tasks/puppet-lint'
 PuppetLint.configuration.send('disable_80chars')
 PuppetLint.configuration.send('disable_140chars')
 PuppetLint.configuration.relative = true
-PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+PuppetLint.configuration.ignore_paths = ['spec/**/*.pp', 'pkg/**/*.pp', 'vendor/**/*.pp']
 
-desc 'Validate manifests, templates, and ruby files'
+desc 'Validate manifests, templates, ruby files and shell scripts'
 task :validate do
-  Dir['manifests/**/*.pp'].each do |manifest|
-    sh "puppet parser validate --noop #{manifest}"
-  end
-  Dir['spec/**/*.rb','lib/**/*.rb'].each do |ruby_file|
+  Dir['spec/**/*.rb', 'lib/**/*.rb'].each do |ruby_file|
     sh "ruby -c #{ruby_file}" unless ruby_file =~ /spec\/fixtures/
   end
-  Dir['templates/**/*.erb'].each do |template|
-    sh "erb -P -x -T '-' #{template} | ruby -c"
+  Dir['files/**/*.sh'].each do |shell_script|
+    sh "bash -n #{shell_script}"
+  end
+end
+
+desc 'Validate that README.md documents all parameters for each class'
```

Comment:
please remove this validate_readme task as we don't need it and the script doesn't exist

Reply:
Sorry, that obviously sneaked in while copy/pasting. Removed it.